### PR TITLE
Add filtering to the API

### PR DESCRIPTION
**Pull request by terceiro**

### CODE REVIEW

File: squad/api/rest.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ class ProjectViewSet(viewsets.ModelViewSet):
     """
     queryset = Project.objects
     serializer_class = ProjectSerializer
+    filter_fields = ('group',)
```
Comment (mwasilew) [inline squad/api/rest.py]: I would add slug and name here

File: squad/api/rest.py [hunk 2]
```diff
@@ -137,6 +138,7 @@ class EnvironmentViewSet(ModelViewSet):
     """
     queryset = Environment.objects
     serializer_class = EnvironmentSerializer
+    filter_fields = ('project',)
```
Comment (mwasilew) [inline squad/api/rest.py]: I would add slug and name here

**Comment by terceiro** [comment]
ok, added those fields and updated the pull request

**Comment by mwasilew** [comment]
Works good enough. For some reason substring search doesn't work for me. According to the docs it should work. Let's merge it and fix the problems later.

**Pull request merged**